### DISCUSSION TITLE
Drasil Research Group Meeting Agenda, 1:30 pm, Friday Mar 11, 2022 (On Teams)

### DISCUSSION THREAD

**Discussion by smiths** [JacquesCarette/Drasil#2950] [Meeting Agendas]
The agenda for this coming week is as follows:

1. Dong update
    - Last week #2947
    - solve ode, projectile. when we can say we solve the problem? generate a program? write it down in pdf?
    - analytical form: expression, equation in paper/ generate a program? projectile is analytical?
    - numerical form: generate a program that can produce concrete results. NoPCM is numerical
    - `codeSpec :: SystemInformation -> Choices -> [Mod] -> CodeSpec` what information be written down, about semantics
    - `Choices` how information be written down
2. Jason update
3. Peter update
4. Theory refinement version of [noPCM](https://github.com/smiths/caseStudies/blob/master/CaseStudies/noPCM/docs/SRS/TheoriesVersion_NoPCM_SRS.pdf)
5. Cold cases:
a. @cd155 #492
b. @peter-michalski #135 
c. @peter-michalski #702
d. @balacij #849
6. Questions/Comments/Concerns/Discussion

**Comment by peter-michalski** [comment]
Added another issue, since #135 might be a quick discussion

**Reply by smiths** [reply]
Great!  (I split the agenda item into two separate items.)
